website/css/comprehensive-demo.css

```
@@ -0,0 +1,932 @@
+    font-size: 1.25rem;
+    text-shadow: 0 2px 4px rgba(0, 0, 0, 0.3);
+}
+
+.chart-subtitle {
+    color: #cbd5e1;
+    font-size: 0.9rem;
+    opacity: 0.8;
+}
+
+.emotion-bars {
+    space-y: 20px;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)